packages/api/test/e2e/promise-alex.spec.ts

```diff
@@ -32,30 +32,31 @@ describe.skip('alex queries', () => {
       console.log('api.query.staking.validators():', res.toJSON());
 
       done();
-    });
+    }).catch();
   });
 
-  it('retrieves a single value', (done) => {
-    api.query.staking.validators('5DuiZFa184E9iCwbh4WjXYvJ88NHvWJbS8SARY8Ev1YEqrri', (res) => {
-      console.error(res);
-
-      console.log('api.query.staking.validators(id):', res.toJSON());
+  describe('retrieves a single value', () => {
+    it('retrieves the list of stash validators', (done) => {
+      api.query.staking.validators('5DuiZFa184E9iCwbh4WjXYvJ88NHvWJbS8SARY8Ev1YEqrri', (res) => {
+        console.error(res);
+        console.log('api.query.staking.validators(id):', res.toJSON());
 
-      done();
+        done();
+      }).catch();
     });
 
     it('Gets the hash of the last finalized header', async (done) => {
       api.rpc.chain.getFinalizedHead((head) => {
         expect(head instanceof Hash).toBe(true);
         done();
-      });
+      }).catch();
```

Comment:
Not on-board with these `.catch()`-es - rather return the promise from the test, ie. `it('...', async () => { return api...`